build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md

```diff
@@ -0,0 +1,223 @@
+# How to mint your own ERC721 on Avalanche using Open Zeppelin
+
+### This tutorial teaches you to deploy your own ERC721 token on Avalanche FUJI C-chain testnet.
+
+### The steps to deploy on the mainnet are identical and the differences are mentioned in the tutorial.
+
+## Contents
```

Comment:
[Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md022)